packages/calypso-e2e/src/browser-helper.ts

```diff
@@ -93,3 +97,34 @@ export function getLaunchConfiguration( chromeVersion: string ): BrowserContextO
 export function targetGutenbergEdge(): boolean {
 	return !! process.env.GUTENBERG_EDGE;
 }
+
+/**
+ * Returns the default Logger configuration.
+ *
+ * The default Logger configuration has the following:
+ * 	- level: pw:api (verbose API logs)
+ * 	- name, severity, message
+ * 	- file name: playwright.log
+ *
+ * @returns {LaunchOptions} Logger configuration.
+ */
+export async function getDefaultLoggerConfiguration(): Promise< LaunchOptions > {
+	const { testPath } = getState() as { testPath: string };
+	const sanitizedTestFilename = path.basename( testPath, path.extname( testPath ) );
+	const resultsPath = path.join( process.cwd(), 'results' );
+	await mkdir( resultsPath, { recursive: true } );
+	const tempDir = await mkdtemp( path.join( resultsPath, sanitizedTestFilename + '-' ) );
```

Comment:
Hmm, it feels a little bit strange to have this temp directory creation in the same function that gets the logger. 

I realize this is starting to sound like whack-a-mole, but could we break this chunk out into a separate function that makes this tmp directory if it doesn't exist, and returns that path, and then that directory path is fed into the logger creating function as an argument?